src/app/wallet/wallet/send/send.component.ts

```diff
@@ -16,27 +15,21 @@ import { AddressLookUpCopy } from '../models/address-look-up-copy';
 
 import { AddressHelper } from '../../../core/util/utils';
 import { TransactionBuilder, TxType } from './transaction-builder.model';
-import { SendConfirmationModalComponent } from 'app/modals/send-confirmation-modal/send-confirmation-modal.component';
-
-// @TODO create/export from seperate file if needed in future?
-class RingSizeConfig {
-  max: number;
-  min: number;
-  constructor(obj: any) {
-    this.max = obj.max;
-    this.min = obj.min
-  }
-}
+import {
+  SendConfirmationModalComponent
+} from 'app/modals/send-confirmation-modal/send-confirmation-modal.component';
 
 @Component({
   selector: 'app-send',
   templateUrl: './send.component.html',
   // TODO merge / globalize styles
-  styleUrls: ['./send.component.scss', '../../settings/settings.component.scss']
+  styleUrls: [
+    './send.component.scss',
+    '../../settings/settings.component.scss'
```

Comment:
Why is Settings SCSS included here? There's no reason for it AFAIK, only creates mess with style inheritance..

Reply:
Its already included in the market branch @AllienWorks.
here: https://github.com/particl/particl-desktop/blob/market/src/app/wallet/wallet/send/send.component.ts#L25.

Can we remove the same from here?

Reply:
The Settings styles were also included in Receive if I'm not mistaken, it was doing a huge styling chaos there (took me a while before I figured it out). So yes, remove it please. If any GFX issues arise, I will fix them. But we shouldn't definitely share component styles across multiple ones, thanks.

Reply:
Thanks, @AllienWorks for confirmation, I'll remove it shortly.